Sources/tinys3/Helpers.swift

```diff
@@ -46,6 +46,12 @@ func sha256Hash(string: String) -> String {
     sha256Hash(data: Data(string.utf8))
 }
 
+func md5Hash(data: Data) -> String {
+    var hasher = Insecure.MD5()
+    hasher.update(data: data)
+    return hasher.finalize().lowercaseHexValue
+}
+
```

Comment:
See https://docs.aws.amazon.com/AmazonS3/latest/userguide/checking-object-integrity.html

When using Multipart uploads, each part's `ETag` is it's md5 checksum (while the ETag for the _whole_ object is not the md5 of the whole file, but instead build from the md5 of a concatenation of each of the part's ETags)